template.yaml

```diff
@@ -485,6 +500,51 @@ Resources:
         - Key: Environment
           Value: !Ref DeploymentEnvironment
 
+  S3Bucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName: !Sub "jfrog-xray-aws-failed-findings-${DeploymentEnvironment}-${AWS::Region}"
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+      LifecycleConfiguration:
+        Rules:
+          - Id: GlacierRule
+            Prefix: glacier
+            Status: Enabled
+            ExpirationInDays: 365
+            Transitions:
+              - TransitionInDays: 30
+                StorageClass: STANDARD_IA
+      Tags:
+        - Key: App
+          Value: xray-aws-security-hub
+        - Key: Environment
+          Value: !Ref DeploymentEnvironment
+
+  BucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref S3Bucket
+      PolicyDocument:
+        Statement:
+          - Action:
+              - 's3:GetObject'
+              - 's3:PutObject'
+            Effect: Allow
+            Principal:
+              AWS:
+                - !GetAtt TransformerRole.Arn
+            Resource:
+              - !Join
+                - ''
+                - - 'arn:aws:s3:::'
```

Comment:
Same here, please scope this to the AWS Account Id

Reply:
It's already scoped to the s3 bucket name. Keeping this.